feather/server/config.toml

```diff
@@ -12,7 +12,7 @@ online_mode = true
 motd = "A Feather server"
 max_players = 16
 default_gamemode = "creative"
-view_distance = 12
+view_distance = 32
```

Comment:
This should probably be reverted? 12 is the default for vanilla too.

Reply:
I thought that the global threadpool can have an unlimited amount of threads? Also, while the channels are blocked, other threads can work in the meantime. And of course, the view_distance should be set to 12.

Reply:
> I thought that the global threadpool can have an unlimited amount of threads?

It's a fixed size AFAIK. See https://docs.rs/rayon-core/1.9.1/rayon_core/struct.ThreadPoolBuilder.html#method.num_threads

> Also, while the channels are blocked, other threads can work in the meantime.

Yes, but the Rayon threads are blocked. Which means other tasks on the Rayon thread pool can't run.

Reply:
Just learned that rayon threads are not the same as os threads :) I am not a fan of making a separate thread for every chunk, do you think a 'personal' ThreadPool would be a good idea?

Reply:
Using `rayon::spawn` won't create a new thread for each chunk. It dispatches a task into a pool of existing threads, which is very cheap.